### PR TITLE
Code review comment

### PR DESCRIPTION
**Pull request by andymc12**
Tweak to PR #12157 - avoid duplicating propertyValue fields by using `super.forEach` and remove trailing comma.

### CODE REVIEW

**Comment by andymc12** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java [hunk 1]
```diff
@@ -1400,10 +1400,11 @@ public String toString() {
             }
         }
         try {
-            forEach((k, v) -> { sb.append(k).append("=").append(v).append(", "); });
+            super.forEach((k, v) -> { sb.append(k).append("=").append(v).append(", "); });
         } catch (ConcurrentModificationException ex) {
             sb.append(" ConcurrentModificationException caught!");
         }
+        sb.setLength(sb.length()-2); //remove trailing comma and space
```
Comment (jhanders34) [inline dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java:1407]: If it there are no properties, then the StringBuffer will only have a { character in it.  As such you would set the length to -1 which will give you a IndexOutOfBoundsException.

**Comment by andymc12** [comment]
New #build for new tweaks and unit test

**Comment by andymc12** [comment]
#run-libby-bot

**Pull request merged**